README.md

```diff
@@ -36,7 +36,7 @@ deploy-code:
 
     steps:
       - name: Download artifacts from release
```

Comment:
Since you have two kind of different paths I think we need another example showing what it is like downloading from a different repository.  That would be a good way to show how you use a different pat and provide the repository argument.